include/aikido/constraint/TSR.hpp

```diff
@@ -112,6 +114,14 @@ class TSR : public Sampleable,
   bool project(const statespace::StateSpace::State* _s,
       statespace::StateSpace::State* _out) const override;
 
+  /// Get the testable tolerance used in isSatisfiable.
+  /// \param[out] _out Testable tolerance, double.
+  double getTestableTolerance();
+  
+  /// Set the testable tolerance used in isSatisfiable.
+  /// \param _testableTolerance Testable tolerance to set.
+  void setTestableTolerance(const double _testableTolerance);
```

Comment:
Don't mark arguments that are passed by value as `const` in function declarations.